### ISSUE TITLE
"/ver", "/version" and "/about" commands issue.

### ISSUE THREAD

**Issue by ghost** [pmmp/PocketMine-MP#6322]
When a player runs "/pl" or "/plugins", they can't see the plugin list because they don't have perms for that. But what's the issue is that when a player runs one of those 3 commands ("/ver", "/version" or "/about") and they type a random letter behind that, they can see all plugins and info about the plugins that contain the letter they write behind that.

**Comment by dktapps** [comment]
Yeah, we probably ought to split up the permissions for this command, or maybe get rid of that feature entirely

**Comment by NhanAZ** [comment]
Why don't we plan for this instead of closing the issue?
Closing an issue instead of planning to resolve it is the new solution of the PMMP team? @Harbor3325

**Comment by ghost** [comment]
they did nothing with that @NhanAZ

**Comment by JavierLeon9966** [comment]
> they did nothing with that @NhanAZ

It doesn't mean they will never fix it. If that was the case, this issue would've been closed by them. Be patient because they not always fix issues immediately.

**Comment by NhanAZ** [comment]
So can I create a new issue similar to this one?

**Comment by nicholass003** [comment]
> So can I create a new issue similar to this one?

yes, if it's not a duplicate

**Comment by nicholass003** [comment]
@Harbor3325 you can use this plugin temporarily https://github.com/nicholass003/VersionCommand
